### PR TITLE
Implement missing `minmax` normalization

### LINKED ISSUE #62: No choices for normalization
`minmax` is available, but not implemented, only `mean` is working

### PR DESCRIPTION
**Pull request by OleMussmann**
Closes #62

### CODE REVIEW

File: src/electricity/electricity.py [hunk 1]
```diff
@@ -54,9 +55,18 @@
...
+    if trainer.normalize == "mean":
+        x_mean = trainer.x_mean.detach().cpu().numpy()
+        x_std = trainer.x_std.detach().cpu().numpy()
+        stats = (x_mean, x_std)
+    elif trainer.normalize == "minmax":
+        x_min = trainer.x_min.detach().cpu().numpy()
+        x_max = trainer.x_max.detach().cpu().numpy()
+        stats = (x_min, x_max)
+    else:
+        raise ValueError("No normalization chosen")
```
Comment (sjvrijn) [inline src/electricity/electricity.py]: A bit nitpicky, but I'd say `f'Invalid normalization option: {trainer.normalize}'` instead. An invalid normalization option is still chosen

**Review by sjvrijn — CHANGES_REQUESTED** [review]

File: src/electricity/electricity.py (hunk 1 above)
Comment (OleMussmann) [reply src/electricity/electricity.py]: Not nitpicky at all, that makes perfect sense. Fixed.

**Review by sjvrijn — APPROVED** [review]
LGTM!
